framework/Metrics/MetricUtilities.py

```diff
@@ -208,7 +209,9 @@ def _getCDFAreaDifference(data1, data2):
   stats1, cdf1, pdf1 =_convertToCommonFormat(data1)
   stats2, cdf2, pdf2 =_convertToCommonFormat(data2)
   low, high = _getBounds(stats1, stats2)
-  return mathUtils.simpson(lambda x:abs(cdf1(x)-cdf2(x)),low,high,100000)
+  #for some earlier tests, simpson was more reliable, but much slower
+  #return mathUtils.simpson(lambda x:abs(cdf1(x)-cdf2(x)),low,high,100000)
```

Comment:
Do you want to keep this commented line?

Reply:
Yes, I want to keep it, just incase someone finds that quad gives pathological behavior.  (I couldn't find the example, but at one point quad was return 0.0 for something that definitely not 0.0, but that was several years ago and with a older version of scipy, so maybe it is fixed)